rnndb/fifo/pcopy.xml

```diff
@@ -57,11 +57,11 @@
 			<bitfield pos="16" name="UNK16">
 				<doc>Always set</doc>
 			</bitfield>
-			<bitfield pos="17" name="SRC_TILE_WIDTH">
+			<bitfield pos="17" name="SRC_BLOCK_WIDTH">
```

Comment:
this is actually GOB width - hence the todo about weird size here, this is the only place that allows other value than 64